### PR TITLE
feat(codebases): adds security alert subscription

### PR DESCRIPTION
**Pull request by invincibleJai**
### Codebases

- adds security alert subscription 

Invision : https://redhat.invisionapp.com/share/VPKMQPFMF8U#/screens/302332662

Tracks : 
- https://github.com/openshiftio/openshift.io/issues/3823

Changes can be merged once other pieces related to notification service is in place

### CODE REVIEW

**Comment by invincibleJai** [comment]
@mindreeper2420 pls review it, when you get time.

File: src/app/space/create/codebases/codebases-item-heading/codebases-item-heading.component.html [hunk 1]
```diff
@@ -22,4 +22,12 @@
       </div>
     </div>
   </div>
+  <div class="list-pf-additional-content">
+    <div>
```
Comment (mindreeper2420) [inline src/app/space/create/codebases/codebases-item-heading/codebases-item-heading.component.html]: This `<div>` is not necessary, as the content is already wrapped in a `<div>`.

File: src/app/space/create/codebases/codebases-item-heading/codebases-item-heading.component.html [hunk 2]
```diff
@@ -22,4 +22,12 @@
...
+      SECURITY ALERT
+      <i class="pficon pficon-info margin-left-5"
+         tooltip="Turn 'ON' the switch to get email notification about CVE(s) in your repo."
```
Comment (mindreeper2420) [inline src/app/space/create/codebases/codebases-item-heading/codebases-item-heading.component.html]: Add an 's' to 'notification'

File: src/app/space/create/codebases/codebases-item/codebases-item.component.html [hunk 3]
```diff
@@ -27,4 +27,11 @@
   <div class="list-pf-additional-content">
```
Comment (mindreeper2420) [inline src/app/space/create/codebases/codebases-item/codebases-item.component.html]: This should be moved inside of the `<ng-template>` and change the class to `list-pf-description`. That will ensure that the content lays out and collapses properly.

There should only be one `<div>` of `'list-pf-additional-content'.

**Comment by invincibleJai** [comment]
> On [Line 11](https://github.com/invincibleJai/fabric8-ui/blob/66f7113b11e4541ed11192024c3fc79f75e4b4a4/src/app/space/create/codebases/codebases-item-heading/codebases-item-heading.component.html#L11) of `codebases-item-heading/codebases-item-heading.component.html`, you need to change that to `list-pf-description` and move it inside of `<div class="list-pf-main-content">`. That way, the content will wrap properly when resized.

This matches the requested change to `codebases-item/codebases-item.component.html`.

@mindreeper2420 i tried that, it's not helping actually , as `list-pf-additional-content` takes lots of space. can see below 
screen shot 2018-07-07 at 10 01 29 am

File: src/app/space/create/codebases/codebases-item-heading/codebases-item-heading.component.html (hunk 1 above)
Comment (invincibleJai) [reply src/app/space/create/codebases/codebases-item-heading/codebases-item-heading.component.html]: This div helped to have title and icon in sync if I remove it, may need to override some styles. Without that `<div>` it's as below

screen shot 2018-07-07 at 10 14 12 am

**Comment by invincibleJai** [comment]
[test]

**Pull request merged**